### PR TITLE
Added credential offer nonce details in issuace data flow

### PR DESCRIPTION
**Pull request by aritroCoder**

### CODE REVIEW

File: spec/data_flow_issuance.md [hunk 1]
```diff
@@ -291,7 +291,7 @@ After the [[ref: issuer]] receives the [[ref: Credential Request]] from the [[re
 
 #### Verifying the Credential Request
 
-Before deciding to issue the credential, the [[ref: issuer]] must first verify the [[ref: Credential Request]] from the [[ref: holder]] by checking first the nonce, and then the blinded link secret correctness proof.
+Before deciding to issue the credential, the [[ref: issuer]] must first verify the [[ref: Credential Request]] from the [[ref: holder]] by using the nonce from credential offer to verify the blinded link secret correctness proof.
```
Comment (swcurran) [inline spec/data_flow_issuance.md:294]: Adjust to "...using the nonce from the credential offer ($n_0$) to verify..." -- "the" and what n_0 is in the following are missing.

File: spec/data_flow_issuance.md [hunk 2]
```diff
@@ -311,13 +311,6 @@ $$ \hat{u} = \tilde{u}$$
 
 
 
-::: todo
```
[the remark this answers is not in the document]
Comment (aritroCoder) [reply spec/data_flow_issuance.md:314]: It is written above the derivation:
image

File: spec/data_flow_issuance.md [hunk 3]
```diff
@@ -243,10 +243,12 @@ The resulting blinded link secret data structure inserted into the [[ref: Creden
...
 
-* `u`: is the blinded link secret which is $(s^v \times A_{bl})\ Mod\ n$.
-* `ur`: is `null` if revocation is not active for the [[ref: Public Credential Definition], and if revocation is active $u_r = h_2^{s'_r}$ where $s'_r$ is randomly selected quadratic residue of order of the bilinear groups `q` and $h_2$ is  part of the revocation public key.
-* `hidden_attributes`: is an array of hidden attributes from the list of [[ref: Public Credential Definition]]. For AnonCreds v1.0, it is always a single entry of `master_secret`.
 
+Where:
+
+* `u`: $u = (s^{v'} \times A_{bl})\ Mod\ n$
+* `ur`: is `null` if revocation is not active for the [[ref: Public Credential Definition], and if revocation is active $u_r = h_2^{s'_r}$ where $s'_r$ is randomly selected quadratic residue of order of the bilinear groups `q` and $h_2$ is  part of the revocation public key.
+* `hidden_attributes`: is an array of hidden attributes from the list of [[ref: Public Credential Definition]. For AnonCreds v1.0, it is always a single entry of `master_secret`.
```
Comment (mikelodder7) [inline spec/data_flow_issuance.md:251]: Are we still using the term master secret?

Comment (aritroCoder) [reply spec/data_flow_issuance.md:251]: I know it's an old term, but what should I use in place of that?

Comment (swcurran) [reply spec/data_flow_issuance.md:251]: Link Secret should always be used except when talking about the name of a data element in a data structure that is used inter-operably across participants -- e.g. in a credential, presentation, credential definition, revocation registry.

Comment (aritroCoder) [reply spec/data_flow_issuance.md]: done

**Review by swcurran — APPROVED** [review]

**Pull request merged**